modules/protect/math-fallback.php

```diff
@@ -83,7 +83,11 @@ static function generate_math_page( $error = false ) {
 		<?php
 			$mathage = ob_get_contents();
```

Comment:
Good catch, I didn't even notice! Fixed in d91cf615bae3d764c880c2694cda5da4abdb2c46